tcga_utils/wsi_utils.py

```diff
@@ -499,16 +499,38 @@ def __init__(self, uuid, mpp=0.5):
         tiles = wsi_utils.slide_to_tiles(slide, new_mpp=mpp)
         filtered_tiles = wsi_utils.filter_tiles(tiles)
         download_utils.remove_file(svs_filename)
-        
+
         self.uuid = uuid
         self.tiles = filtered_tiles
         self.mpp = mpp
         self.annotation = {}
         return None
 
+    def tcga_dictionary():
```

Comment:
I think aspects of the dataset (eg where the actual table) should be stored in the class itself. I.e. you should create a self.tcga_df in __init__